tests/Microsoft.Windows.Developer/Microsoft.Windows.Developer.Tests.ps1

```diff
@@ -145,6 +145,34 @@ Describe 'UserAccessControl'{
    }
 }
 
+Describe 'Animation'{
+   It 'Keeps current value.'{
+      $initialState = Invoke-DscResource -Name AnimationEffects -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+
+      $parameters = @{ AnimationBehavior = 'KeepCurrentValue' }
+
+      $testResult = Invoke-DscResource -Name AnimationEffects -ModuleName Microsoft.Windows.Setting.Accessibility -Method Test -Property $parameters
+      $testResult.InDesiredState | Should -Be $true
+      
+      # Invoking set should not change these values.
+      Invoke-DscResource -Name AnimationEffects -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $parameters
+      $finalState = Invoke-DscResource -Name AnimationEffects -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+      $finalState.AnimationBehavior | Should -Be $initialState.AnimationBehavior
+   }
+
+   It 'Sets desired value.'{
+      # Randomly generate desired state. Minimum is set to 1 to avoid using KeepCurrentValue
+      $desiredAnimationBehavior = [AnimationBehavior]("Enabled","Disabled"|Get-Random)
+
+      $desiredState = @{ AnimationBehavior = $desiredAnimationBehavior }
+      
+      Invoke-DscResource -Name AnimationEffects -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $desiredState
+   
+      $finalState = Invoke-DscResource -Name AnimationEffects -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+      $finalState.AnimationBehavior | Should -Be $desiredAnimationBehavior

```

Comment:
Add to these tests to check if the value you are setting actually matches what is returned from the get method. This check needs to happen for all of the DSC Properties